spark2/src/test/java/org/apache/iceberg/spark/source/TestSparkSchema24.java

```diff
@@ -19,5 +19,43 @@
 
 package org.apache.iceberg.spark.source;
 
+import java.io.IOException;
+import java.util.List;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.SnapshotSummary;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.hadoop.HadoopTables;
+import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.util.PropertyUtil;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.Assert;
+import org.junit.Test;
+
 public class TestSparkSchema24 extends TestSparkSchema {
+
+  @Test
+  public void testReaderUtils() throws IOException {
```

Comment:
I don't think this test needs to be here. Tests for utility classes should be in their own files, and preferably in the same module as the utility class.

Reply:
ack